src/Payments.sol

```diff
@@ -795,6 +794,7 @@ contract Payments is
         public
         nonReentrant
         validateRailActive(railId)
+        onlyRailOperator(railId)
```

Comment:
Why? I think we definitely want any party to be able to do this. We can restrict it to client + operator + SP, but we don't want to wait on the operator.